index.bs

```diff
@@ -592,7 +592,6 @@ A [=sensor=] has an associated <dfn>current polling frequency</dfn> which is ini
 <pre class="idl">
 [SecureContext]
 interface Sensor : EventTarget {
-  readonly attribute SensorState state;
```

Comment:
Will it be modified into `boolean activated`, like you and @rwaldron discussed at https://github.com/w3c/sensors/issues/160#issuecomment-291259983  or we'll leave it not exposed for now?